src/mapgen/treegen.cpp

```diff
@@ -32,6 +32,16 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 namespace treegen
 {
 
+void TreeDef::resolveNodeNames()
+{
+	getIdFromNrBacklog(&trunknode.param0, "", CONTENT_IGNORE, false);
+	getIdFromNrBacklog(&leavesnode.param0, "", CONTENT_IGNORE, false);
+	if(leaves2_chance)
+		getIdFromNrBacklog(&leaves2node.param0, "", CONTENT_IGNORE, false);
+	if(fruit_chance)
+		getIdFromNrBacklog(&fruitnode.param0, "", CONTENT_IGNORE, false);
```

Comment:
error_on_fallback=true
```suggestion
	getIdFromNrBacklog(&trunknode.param0, "", CONTENT_IGNORE);
	getIdFromNrBacklog(&leavesnode.param0, "", CONTENT_IGNORE);
	if (leaves2_chance)
		getIdFromNrBacklog(&leaves2node.param0, "", CONTENT_IGNORE);
	if (fruit_chance)
		getIdFromNrBacklog(&fruitnode.param0, "", CONTENT_IGNORE);
```